src/content/docs/pl/install/auto.mdx

```diff
@@ -10,6 +10,7 @@ Gotów do instalacji Astro? Podążaj za tym poradnikiem korzystając z CLI `cre
 :::note[Wolisz zainstalować Astro ręcznie?]
 Przeczytaj zamiast tego naszą [instrukcję instalacji krok po kroku](/pl/install/manual/).
 :::
+
```

Comment:
The content of the English version hasn't changed - it's just been moved to a new location and this file shows up as "needs changes". I think that adding one missing new line will solve this problem.

Reply:
Here you can see the changes made to the file before it was moved, I believe the only missing change is the updated Liftoff message, could you take a look at this?

https://github.com/withastro/docs/commits/19fa02fbe282556a2747f1db79933bea28f9ddea/src/pages/en/install/auto.mdx?browsing_rename_history=true&new_path=src/content/docs/en/install/auto.mdx&original_branch=main

Reply:
Yes. Also, I’m not sure if we should translate the CLI output as has been done in this document. Currently the CLI always outputs in English so it might be more helpful to leave those in English? (Perhaps followed by a translation in parentheses?)

Reply:
When I get home I will, of course, correct both pages. I'll ping you on Discord when I'm done ❤️